ratbagd/ratbagd-profile.c

```diff
@@ -291,8 +291,7 @@ static int ratbagd_profile_set_active(sd_bus_message *m,
 		return r;
 
 	r = ratbag_profile_set_active(profile->lib_profile);
-	if (r < 0)
-		return r;
+	assert(r == 0);
```

Comment:
I am not sure I read @whot's comment like that in #316:
To me, the assert would be on the errors after the call of ratbag_*, not the error on the call of ratbag_*. Because here you assume the device will always play nice, and a disconnection for instance while you are calling the dbus method would make the whole ratbagd abort.

I think we should not assert on libratbag errors, but only for dbus like errors, which we already don't check as a matter of fact.

Reply:
yep, agree. anything that's a dbus error is a bug in ratbagd or a system failure we don't recover from. A ratbag error means something else is wrong and should trigger a signal for the client to re-sync.